Dockerfile.mayhem

```diff
@@ -0,0 +1,18 @@
+FROM --platform=linux/amd64 ubuntu:22.04 as builder
```

Comment:
Just wondering if the sed commands are necessary? Can we just set these env vars to false when running the bootstrap script, rather than modifying the script itself?

Reply:
Unfortunately, these are not environment variables; they are Python variables. The bootstrap script `echo`es lines of Python code to a Python configuration script. 

I reduced the number of `sed` calls, but this one is necessary for the build to succeed.